docs/hosted.rst

```diff
@@ -43,6 +43,19 @@ If you're running analysis operations, SSH in as the *analysis* user::
 
 Once logged in, you can take advantage of the powerful server to carry out analysis operations, such as using flatten-tool on files, more quickly than on your local machine. The analysis user has read-only access to the files downloaded by the scrapers. Please remember to delete your files when you're done! 
 
+The tools available are:
+
+* jq
+* flatten-tool. This is in /home/analysis/flatten-tool and there is a virtual environment in `.ve`.
+* ocdskit. This is in /home/analysis/ocdskit and there is a virtual environment in `.ve`.
```

Comment:
Somewhat overlapping with https://github.com/open-contracting/deploy/issues/38, should these instead be available system-wide? In another issue @duncandewhurst described that he pip installed ocdskit, as he couldn't figure out how to run it the way it was currently installed.